stackslib/src/chainstate/nakamoto/coordinator/tests.rs

```diff
@@ -282,9 +282,7 @@ fn test_simple_nakamoto_coordinator_bootup() {
 /// Mine a single Nakamoto tenure with 10 Nakamoto blocks
 #[test]
 fn test_simple_nakamoto_coordinator_1_tenure_10_blocks() {
-    let mut test_signers = TestSigners::default();
-    let mut peer = boot_nakamoto(function_name!(), vec![], test_signers.aggregate_public_key);
-    let private_key = peer.config.private_key.clone();
+    let private_key = StacksPrivateKey::new();
```

Comment:
Yep -- I'd recommend using a constant private key, e.g.:

```suggestion
    let private_key = StacksPrivateKey::from_seed(&[2]);
```